src/test/java/com/hartwig/snpcheck/SnpCheckTest.java

```diff
@@ -52,6 +56,14 @@
     private static final String BARCODE = "barcode";
     private static final Sample REF_SAMPLE = new Sample().barcode(BARCODE).name("sampler");
     private static final Sample TUMOR_SAMPLE = new Sample().barcode(BARCODE).name("samplet");
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
bit of duplication here, maybe a shared util to create the OM?

Reply:
Fair enough, cleaned up.